cmd/common.go

```diff
@@ -237,6 +238,64 @@ func loadConfig(nodetype check.NodeType) string {
 	return filepath.Join(path, file)
 }
 
+func mapToCISVersion(kubeToCISMap map[string]string, kv string) string {
+	cisVersion, found := kubeToCISMap[kv]
+	for !found && (kv != defaultKubeVersion && !isEmpty(kv)) {
+		kv = decrementVersion(kv)
+		cisVersion, found = kubeToCISMap[kv]
+		glog.V(2).Info(fmt.Sprintf("mapToCISVersion for cisVersion: %q found: %t\n", cisVersion, found))
+	}
+
+	if !found {
+		glog.V(2).Info(fmt.Sprintf("mapToCISVersion unable to find a match for: %q", kv))
```

Comment:
```suggestion
		glog.V(1).Info(fmt.Sprintf("mapToCISVersion unable to find a match for: %q", kv))
```
Since it's an error case I would make this log at level 1